src/config.js

```diff
@@ -1,16 +1,26 @@
-exports.ORG_WHITELIST = ['preset-io', 'Polidea', 'polidea', 'airbnb', 'dropbox', 'lyft', 'Flexiana', 'Turing'];
+exports.ORG_WHITELIST = [
+  "preset-io",
+  "Polidea",
+  "polidea",
+  "airbnb",
+  "dropbox",
+  "lyft",
+  "Flexiana",
+  "Turing",
+  "Superset-Community-Partners",
```

Comment:
Does the org show up like this with hyphens when it's returned by git?

Reply:
@AAfghahi and I curled the api and verified that it returns the name with hyphens.